### ISSUE TITLE
Can't download attachment with special character in file name

### ISSUE THREAD

**Issue by nuali** [wefork/wekan#97]
Hey,

when I try to download an attachment from a card which has a special character in the file name (in my case an Ü), I get the following error:

```
TypeError: The header content contains invalid characters
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:363:13)
    at ServerResponse.res.setHeader (/programs/server/npm/node_modules/meteor/webapp/node_modules/connect/lib/patch.js:134:22)
    at Proxy.translateResponse (server/proxy.js:1779:18)
    at server/proxy.js:2016:21
    at /programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:32:39
```

The reason seems to be an node.js update: 
https://www.alexkras.com/typeerror-the-header-content-contains-invalid-characters/

Can you please help me with that problem? :)

Thanks and sunny greetings!

**Comment by nuali** [comment]
Sorry, I checked the closed issues but somehow I missed this one: https://github.com/wefork/wekan/issues/87

**Comment by xet7** [comment]
@nuali 

Do you have newest version of Wekan? Are you using MongoDB 3.2.x ?

I think this bug is already fixed.

**Comment by nuali** [comment]
@xet7 
I'm using it with sandstorm.

**Comment by xet7** [comment]
@nuali 

[Wefork has been merged back to Wekan](https://github.com/wekan/wekan/wiki/FAQ#what-was-wekan-fork--wefork).

Please wait that new Wekan version release [is made for Sandstorm, it's in progress](https://github.com/wekan/wekan/issues/799), and it will include fixes, new features.etc.

**Comment by nuali** [comment]
@xet7 
Alright, thanks, I just used the issue button inside the app, this is how I came here. :)

**Comment by ocdtrekkie** [comment]
@nuali That link will get fixed next time a release is made for the Sandstorm version as well. :)

**Comment by nuali** [comment]
Actually, it wasn't fixed in the newest release 0.47.0~2017-10-04 :(

**Comment by xet7** [comment]
Fixed already.
